### PR TITLE
Add classification of SQL_Type: this PR is just for DML

### PR DESCRIPTION
**Pull request by zhang1002**
Add classification of SQL_Type: this PR is just for DML

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zhang1002** [comment]
cc @yaooqinn @ulysses-you

File: dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json [hunk 1]
```diff
@@ -26,5 +26,9 @@
   "DropTableCommand": "ddl",
   "TruncateTableCommand": "ddl",
   "AlterTableRecoverPartitionsCommand": "ddl",
-  "SetCatalogAndNamespace": "ddl"
+  "SetCatalogAndNamespace": "ddl",
```
Comment (yaooqinn) [inline dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json:29]: this seems not a ddl?

**Review by yaooqinn — APPROVED** [review]
Hope you will fix the test latter, they look ugly to me

**Comment by yaooqinn** [comment]
thanks, merged to master

File: dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json (hunk 1 above)
Comment (zhang1002) [reply dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json:29]: the "SetCatalogAndNamespace" is the grammer of USE's simpleName.
In spark_version_3.1.2, this grammar is belong to Data Definition Statements.

Comment (yaooqinn) [reply dev/kyuubi-extension-spark-3-1/src/test/resources/sql-classification-default.json:29]: got it

**Pull request closed without merging**